lib/generators/manageiq/plugin/templates/Gemfile

```diff
@@ -0,0 +1,12 @@
+# Declare your gem's dependencies in <%= plugin_name %>.gemspec.
+# Bundler will treat runtime dependencies like base dependencies, and
+# development dependencies will be added by default to the :development group.
+gemspec
+
+# Declare any dependencies that are still in development here instead of in
+# your gemspec. These might include edge Rails or gems from your path or
+# Git. Remember to move these dependencies to your gemspec before releasing
+# your gem to rubygems.org.
+
+# Load Gemfile with dependencies from manageiq
+eval_gemfile(File.expand_path("spec/manageiq/Gemfile", __dir__))
```

Comment:
@jrafanie Note that I reverted to using eval_gemfile.  It turns out that if you put bundler-inject here then the definition in the spec/manageiq collides.